dynamiqs/plot/wigner.py

```diff
@@ -30,8 +30,18 @@ def _plot_wigner_data(
     cross: bool = False,
     clear: bool = False,
 ):
+    r"""Plot the Wigner function for pre-computed Wigner data.
+
+    Note:
+        See [`dq.plot.wigner()`][dynamiqs.plot.wigner] for more details.
```

Comment:
How about the "See Also" admonition here? Could be quite relevant to link to both `dq.plot.wigner` and `dq.wigner`.